PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -171,6 +171,12 @@ then
     #fix this if the script is more than one level below HCPPIPEDIR
     export HCPPIPEDIR="$(dirname -- "$0")/.."
 fi
+# parameter initialization
+InitDof="${InitDof:=""}"
+TXwTemplateBrain="${TXwTemplateBrain:=""}"
+OutputTXwImageSTRING="${OutputTXwImageSTRING:=""}"
+CustomMask="${CustomMask:=""}"
+betbiasfieldcor="${betbiasfieldcor:=""}"
```

Comment:
Is this temporary?  We don't use environment variables to pass pipeline-specific settings in any other pipelines.